packages/client/src/query/WhereClause.ts

```diff
@@ -32,7 +32,8 @@ export type PossibleWhereClauseFilters =
   | "$lt"
   | "$lte"
   | "$within"
-  | "$intersects";
+  | "$intersects"
+  | "$startsWith";
```

Comment:
Its kind of wild that we didn't need to write any additional converter code for this to work.

Reply:
Yea I was surprised too